cmd/kops/create_cluster.go

```diff
@@ -244,7 +249,7 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 			subnet := &cluster.Spec.Subnets[i]
 			existingSubnets[subnet.Name] = subnet
 		}
-		for _, zoneName := range parseZoneList(c.Zones) {
+		for _, zoneName := range parseInputList(c.Zones) {
```

Comment:
If you make the var a []string, you don't have to rename (or reuse) parseZoneList I don't think